### DISCUSSION TITLE
Firejail for only Zoom, nothing else

### DISCUSSION THREAD

**Discussion by bouncinglime** [netblue30/firejail#4553] [Q&A] [answered]
I only want to use firejail on zoom. I cannot find a way in the documentation to remove it from all the other programs. It has broken several things, including my password manager browser integration. The last time I used firejail on vanilla Ubuntu, I had to explicitly call it when I wanted to use it. I want that behavior again.

### Environment

`lsb_release -a`
No LSB modules are available.
Distributor ID:	Pop
Description:	Pop!_OS 21.04
Release:	21.04
Codename:	hirsute

`firejail --version`
firejail version 0.9.64.4

Compile time support:
	- AppArmor support is enabled
	- AppImage support is enabled
	- chroot support is enabled
	- D-BUS proxy support is enabled
	- file and directory whitelisting support is enabled
	- file transfer support is enabled
	- firetunnel support is enabled
	- networking support is enabled
	- overlayfs support is disabled
	- private-home support is enabled
	- private-cache and tmpfs as user enabled
	- SELinux support is enabled
	- user namespace support is enabled
	- X11 sandboxing support is enabled

**Comment by rusty-snake** [comment, marked as answer]
If you ran `sudo firecfg`, run `sudo firecfg --clean` to undo [1]. Then you can use one of the solutions from https://github.com/netblue30/firejail/issues/3665#issuecomment-707689049 to sandbox zoom, e.g. `sudo ln -s /usr/bin/firejail /usr/local/bin/zoom`.

\[1]: <https://github.com/netblue30/firejail/wiki/Frequently-Asked-Questions#how-do-i-undo-firecfg>

**Reply by bouncinglime** [reply]
Thank you so much! ^_^ That seems to be working.

It would be great if that question could be expanded to include information about how to set up firejail for specific applications. I had seen that answer, but didn't think it would apply because I don't want to "undo firejail" I want to keep it but only for specific applications.
